TMDBApp/TMDBApp/Language.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  Language.swift
+//  TMDBApp
+//
+//  Created by Richard Leh on 17/01/2017.
+//  Copyright © 2017 Richard Leh. All rights reserved.
+//
+
+import Foundation
+
+struct Language{
+    static let defaulUserKey = "defaulLanguage"
+    static let appResult = ["en-US", "pt-BR", "es-ES", "hr-HR", "ja-JP"]
+    
+    static func setDefaultLanguage(_ lang:String){
```

Comment:
Hmm maybe using something like this for get. But I'm not so sure.
```
static var language:String = {
    let defaults = UserDefaults.standard
    guard let lang = defaults.value(forKey: Language.defaulUserKey) as? String else{
        return ""
    }
     return lang
}()
```
And for set I can't think a better way.